### DISCUSSION TITLE
Lost my tree nodes/lines

### DISCUSSION THREAD

**Discussion by andwan0** [gitextensions/gitextensions#8985] [Q&A]
Somehow the tree node/lines in the main explorer view... is gone/missing.
Now the explorer view looks like a normal table list...

**Comment by RussKie** [comment]
Screenshots?

**Comment by mstv** [comment]
Does it reappear after refreshing (hit <kbd>F5</kbd>)?
Report this as a bug, please, and fill in the template.

**Reply by RussKie** [reply]
I think we have a setting to hide lanes

**Reply by pmiossec** [reply]
> I think we have a setting to hide lanes

Yes. Check  "View"  > "Show revision graph column"
